prev/cli.js

```diff
@@ -288,22 +271,9 @@ function normalizeConfig(config) {
 
 async function queueRestart() {
   const mod = await import('./kernel/index.js')
```

Comment:
Should I remove direct import from here and add `getServer` function in config?

Reply:
Nope